templates/repo/issue/view_content.tmpl

```diff
@@ -44,18 +44,7 @@
 							{{end}}
 						</div>
 						<div class="comment-header-right actions gt-df gt-ac">
-							{{if gt .Issue.ShowRole 0}}
-								{{if (.Issue.ShowRole.HasRole "Writer")}}
-									<div class="ui basic label role-label">
-										{{$.locale.Tr "repo.issues.collaborator"}}
-									</div>
-								{{end}}
-								{{if (.Issue.ShowRole.HasRole "Owner")}}
-									<div class="ui basic label role-label">
```

Comment:
There were "role-label" classes, they are lost now?

Reply:
Only in this place have `role-label` class, didn't understand the usage here, but I understand the usage of `role-label` now: hide them in mobiles.

In GitHub will not display role label in mobiles, so we can add this in the general template.
![image](https://github.com/go-gitea/gitea/assets/18380374/1ee49419-5bca-45a9-a7ef-fd2fa8fa49ba)
![image](https://github.com/go-gitea/gitea/assets/18380374/c5d4d27d-955d-4b93-85b8-58dd2391a5d4)

In Gitea:
![image](https://github.com/go-gitea/gitea/assets/18380374/bb8ce6d3-1b7c-424a-87da-6faf09d3f7f3)
After:
![image](https://github.com/go-gitea/gitea/assets/18380374/6fb3cf93-b526-43e2-aa58-dbac9fe8af34)